exercises/trampoline/solution/solution.js

```diff
@@ -1,19 +1,20 @@
 function repeat(operation, num) {
   return function() {
-    if (num <= 0) return
-    operation()
-    return repeat(operation, --num)
-  }
+    if (num <= 0) { return; }
+    operation();
+    return repeat(operation, --num);
+  };
 }
 
 function trampoline(fn) {
   while(fn && typeof fn === 'function') {
-    fn = fn()
+    return fn();
```

Comment:
This… just calls the function, so it's no longer a trampoline. i.e. why even have the while loop around it?